### ISSUE TITLE
Error `invalid type "age1yubikey"` with current Go Age

### ISSUE THREAD

**Issue by catleeball** [str4d/age-plugin-yubikey#43]
## Environment

* OS: Fedora 35, linux 5.15.5
* age-plugin-yubikey version:  0.2.0
* age version: 1.0.0
* rage version: 0.7.0

## What were you trying to do

- Generated a new age key
- Tried encrypting some sample data
- Encryption works with rage 0.7.0
- Encryption fails with age 1.0.0 with error `age: error: malformed recipient "age1yubikey1qgqsp5z24ff2wkezx0x8vdsv7hwywhj68tp9m6hfchjuunlx8h5pzx33eal": invalid type "age1yubikey"`

## What happened

- Generated key successfully

```fish
> age-plugin-yubikey
✨ Let's get your YubiKey set up for age! ✨

[...]

🔑 Select a YubiKey: Yubico YubiKey OTP+FIDO+CCID 00 00 (Serial: 123)
🕳️  Select a slot for your age identity: Slot 20 (Empty)
📛 Name this identity [age identity TAG_HEX]: AGE_ENC_20
🔤 Select a PIN policy: Once   (A PIN is required once per session, if set)
👆 Select a touch policy: Always (A physical touch is required for every decryption)
Generate new identity in slot 20? yes


Enter PIN for YubiKey with serial 123 (default is 123456): [hidden]

✨ Your YubiKey is using the default management key.
✨ We'll migrate it to a PIN-protected management key.
... Success!
👆 Please touch the YubiKey

📝 File name to write this identity to: age-yk-id-20.txt

✅ Done! This YubiKey identity is ready to go.

🔑 Here's your shiny new YubiKey recipient:
  age1yubikey1qgqsp5z24ff2wkezx0x8vdsv7hwywhj68tp9m6hfchjuunlx8h5pzx33eal

Here are some example things you can do with it:

- Encrypt a file to this identity:
  $ cat foo.txt | age -r age1yubikey1qgqsp5z24ff2wkezx0x8vdsv7hwywhj68tp9m6hfchjuunlx8h5pzx33eal -o foo.txt.age

- Decrypt a file with this identity:
  $ cat foo.txt.age | age -d -i age-yk-id-20.txt > foo.txt

- Recreate the identity file:
  $ age-plugin-yubikey -i --serial 123 --slot 20 > age-yk-id-20.txt

- Recreate the recipient:
  $ age-plugin-yubikey -l --serial 123 --slot 20

💭 Remember: everything breaks, have a backup plan for when this YubiKey does.


> cat age-yk-id-20.txt 
#       Serial: 123, Slot: 20
#         Name: AGE_ENC_20
#      Created: Sat, 04 Dec 2021 05:17:26 +0000
#   PIN policy: Once   (A PIN is required once per session, if set)
# Touch policy: Always (A physical touch is required for every decryption)
#    Recipient: age1yubikey1qgqsp5z24ff2wkezx0x8vdsv7hwywhj68tp9m6hfchjuunlx8h5pzx33eal
```

- Tried encrypting a test string, failed to encrypt with age 1.0.0

```fish
> echo "testo testo" | age -r age1yubikey1qgqsp5z24ff2wkezx0x8vdsv7hwywhj68tp9m6hfchjuunlx8h5pzx33eal -o test.txt.age
age: error: malformed recipient "age1yubikey1qgqsp5z24ff2wkezx0x8vdsv7hwywhj68tp9m6hfchjuunlx8h5pzx33eal": invalid type "age1yubikey"
age: report unexpected or unhelpful errors at https://filippo.io/age/report
```

- Tried encrypting and decrypting with rage 0.7.0, success

```
[21:38:32] cat@ /home/cat/.config/age
> echo "testo testo" | rage -r age1yubikey1qgqsp5z24ff2wkezx0x8vdsv7hwywhj68tp9m6hfchjuunlx8h5pzx33eal -o test.txt.age

[21:39:14] cat@ /home/cat/.config/age
> cat test.txt.age 
age-encryption.org/v1
-> piv-p256 /Ghj/w A2WpYUuG0RnqNqzUyDrGq55lXTCuaR5dqtM19P4OK2DH
jVlT9ZsqZ73zs5C3bS6I2U5gYISwR8jFfXDAODQbt6s
-> d,`K%5-grease
++416eYz7ZkGoYhQTcUsdcqPoeI
--- O2mTiDy+DNkhd4C4ajqYITKmXDJAMiD5TpnG48P48ec
�ɟ)��׬ vBE�|ɪL�c^B���
                     롡g��x�i<z⏎                                                                 

[21:39:17] cat@ /home/cat/.config/age  
> cat test.txt.age | rage -d -i age-yk-id-20.txt > test.txt 

[21:40:11] cat@ /home/cat/.config/age  
> cat test.txt
testo testo
```

## Notes

- Please let me know if there's any debug info I can provide to help!

**Comment by str4d** [comment]
I think this is because Go age 1.0 doesn't have plugin support. IIRC @FiloSottile was leaving it to the 1.1 release.

**Comment by catleeball** [comment]
Oh, I didn't know that! I'll keep an eye out for Go age 1.1 and use rage for my yubikey credential in the meantime.

Thanks for the info! :)

**Comment by str4d** [comment]
Plugin support just landed in Go age v1.1.0-rc1: https://github.com/FiloSottile/age/releases/tag/v1.1.0-rc.1

**Comment by catleeball** [comment]
Nice!! Thanks for the update! :)
